Gemfile

```diff
@@ -30,6 +30,7 @@ gem 'sidekiq'
 gem 'puma'
 
 # Use SCSS for stylesheets
+# NOTE: dependency gem sass-rails => gem sassc => native `libsass` which may take minutes to compile
```

Comment:
Maybe we don't need this gem either?
I cannot find any sass/sass file in the project. Rules in `app/assets/stylesheets` are regular `.css`. Third-party libs (e.g. `bootstrap`) live in `vendor/assets` and rely on their own stylesheets with their own builds.

Reply:
we do have `*.scss` files in this project 
![image](https://user-images.githubusercontent.com/1402651/124187323-88222c80-da72-11eb-9f2e-4b4723fed0c1.png)

I'm not inclined to spend time to test removing `sass-rails` until we experience a practical problem with it.